src/components/organisms/SideNav/index.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import SideNav from '.'
+
+export default {
+  title: 'organisms/SideNav',
+  component: SideNav,
+  argTypes: {},
+} as ComponentMeta<typeof SideNav>
+
+const Template: ComponentStory<typeof SideNav> = (args) => <SideNav {...args} />
+
+export const sideNav = Template.bind({})
```

Comment:
For SideNav Component, 
- In Constants, Create interface for NavItems
           - nested item will also have route, icon can be optional
- SideNav Data can be sent as props as well, have condition in useEffect, if it not available in props it can be loaded from constants

Reply:
done